ardupilot/equipment/power/20010.SmartBatteryCells1.uavcan

```diff
@@ -0,0 +1,9 @@
+#
+# Smart battery cell voltages & resistances
+# Rate: set by parameter on smart battery (default off)
+#
+
+float16[<=24] voltages            # [mV]
+float16[<=24] resistances         # [m Ohm] cell resistances: NAN first element: field not provided

```

Comment:
Coming back to this I'm thinking it might be best to split resistances off as a separate message?